core/src/avm2/globals/flash/display/Stage.as

```diff
@@ -0,0 +1,113 @@
+package flash.display {
+    import flash.accessibility.AccessibilityProperties;
+    import flash.filters.BitmapFilter;
+    import flash.geom.Rectangle;
+    import flash.geom.Transform;
+    import flash.ui.ContextMenu;
+    import flash.system.LoaderInfo;
+
+    [Ruffle(NativeInstanceInit)]
+    public class Stage extends DisplayObjectContainer {
+
+        public function Stage() {
+            throw new Error("You cannot construct new instances of the Stage.")
+        }
+
+        public native function set accessibilityProperties(value:AccessibilityProperties):void;
+
+        override public native function set alpha(value:Number):void;
+
+        override public native function set blendMode(value:String):void;
+
+        override public native function set cacheAsBitmap(value:Boolean):void;
+
+        override public native function set contextMenu(value:ContextMenu):void;
+
+        override public native function set filters(value:Array):void;
+
+        override public native function set focusRect(value:Object):void;
+
+        public native function set loaderInfo(value:LoaderInfo):void;
+
+        override public native function set mask(value:DisplayObject):void;
+
+        override public native function set mouseEnabled(value:Boolean):void;
+
+        override public native function get name():String;
+        override public native function set name(value:String):void;
+
+        override public native function set opaqueBackground(value:Object):void;
+
+        override public native function set rotation(value:Number):void;
+
+        public native function set scale9Grid(value:Rectangle):void;
```

Comment:
This should be an override, I believe.

Reply:
Actually, it should also be implemented (or at least stubbed) on `DisplayObject`, otherwise this will `VerifyError` if it is added as an override. Pure Rust-defined classes do not `VerifyError`, whih is why the problem did not show up previously. For correct behavior, though, this needs to be stubbed on `DisplayObject`. The same problem is present with `loaderInfo` and `accessibilityProperties`, which need to be marked as `override` and stubbed in `DisplayObject`.

Reply:
We should eventually implement them on DisplayObject, but it's not really related to this PR - trying to call on on DisplayObject will give you an error both before and after this PR.